pkgs/tools/system/sleuthkit/default.nix

```
@@ -1,9 +1,47 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, libewf, afflib, openssl, zlib }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, libewf, afflib, openssl, zlib, openjdk, perl, ant }:
 
 stdenv.mkDerivation rec {
   version = "4.11.0";
   pname = "sleuthkit";
 
+  rdeps = stdenv.mkDerivation rec {
+    # Fetch libraries using a fixed output derivation
+
+    version = "1.0";
+    pname = "sleuthkit-deps";
+    nativeBuildInputs = [ autoreconfHook openjdk ant ];
+
+    src = fetchFromGitHub {
+        owner = "sleuthkit";
+        repo = "sleuthkit";
+        rev = "${pname}-${version}";
+        sha256 = "sha256:1dh06k8grrj3wcx3h9m490p69bw41dz6cv8j5j1drpldmv67k3ki";
+    };
+
+    IVY_HOME = "/build/.ant";
+    phases = "unpackPhase buildPhase installPhase";
+
+    buildPhase = ''
+        pushd bindings/java
+        ant retrieve-deps
+        popd
+        pushd case-uco/java
+        ant get-ivy-dependencies
+        popd
+    '';
+
+    installPhase = ''
+        mkdir -m 755 -p $out/bindings/java
+        cp -r bindings/java/lib $out/bindings/java
+        mkdir -m 755 -p $out/case-uco/java
+        cp -r case-uco/java/lib $out/case-uco/java
+        cp -r $IVY_HOME/lib $out
+        chmod -R 755 $out/lib
```

```suggestion
      mkdir -m 755 -p $out/bindings/java
      cp -r bindings/java/lib $out/bindings/java
      mkdir -m 755 -p $out/case-uco/java
      cp -r case-uco/java/lib $out/case-uco/java
      cp -r $IVY_HOME/lib $out
      chmod -R 755 $out/lib
```
Please format the file with nixpkgs-fmt.

---

Done